### PR TITLE
KTOR-6303 Migrate client plugins to new API

### PR DESCRIPTION
**Pull request by rsinukov**

### CODE REVIEW

**Comment by e5l** [comment]
Please check the CI build

File: ktor-client/ktor-client-plugins/ktor-client-logging/common/src/io/ktor/client/plugins/logging/Logging.kt [hunk 1]
```diff
@@ -20,86 +22,95 @@ private val ClientCallLogger = AttributeKey<HttpClientCallLogger>("CallLogger")
 private val DisableLogging = AttributeKey<Unit>("DisableLogging")
 
 /**
- * A client's plugin that provides the capability to log HTTP calls.
- *
- * You can learn more from [Logging](https://ktor.io/docs/client-logging.html).
+ * A configuration for the [Logging] plugin.
```
[the remark this answers is not in the document]
Comment (vnikolova) [reply ktor-client/ktor-client-plugins/ktor-client-logging/common/src/io/ktor/client/plugins/logging/Logging.kt:25]: Do we need to update the Logging docs? The link looks correct.

[the remark this answers is not in the document]
Comment (rsinukov) [reply ktor-client/ktor-client-plugins/ktor-client-logging/common/src/io/ktor/client/plugins/logging/Logging.kt:25]: The link is in the `val Logging` declaration, same as it was.

[the remark this answers is not in the document]
Comment (rsinukov) [reply ktor-client/ktor-client-plugins/ktor-client-logging/common/src/io/ktor/client/plugins/logging/Logging.kt:25]: @vnikolova This change is source-compatible, so nothing to update in the docs.

**Review by e5l — APPROVED** [review]

**Pull request merged**